Documentation/torcx.md

```diff
@@ -0,0 +1,67 @@
+# Torcx configuration managed by Tectonic
```

Comment:
torcx is always lowercase, right?

Reply:
It doesn't have any strong casing. `tectonic-torcx` is always used lowercase as a binary name, while torcx follows normal casing rules depending on the context as it refers to a mix of project, binary, and general addon framework.